sdk/template/aztemplate/ci.yml

```diff
@@ -0,0 +1,19 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  paths:
+    include:
+    - sdk/template/aztemplate/
+    - eng/
```

Comment:
Do we actually want changes under eng to trigger this pipeline? In most languages we leave that up to validating under the core pipeline.

Reply:
@weshaggard This package exists as a way to validate automation does not break Go.  Do you prefer to not trigger this package for changes in Eng?

Reply:
In other languages we usually only include the core pipeline to validate changes under eng. If you add it here then we end up copying that to all the new pipelines and then everyone does it which is not something I think we want.